app/services/workflow_get_service.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'mixins/group_validate_mixin'
+
+class WorkflowGetService
+  include GroupValidateMixin
+  attr_accessor :workflow_id
+
+  def initialize(workflow_id)
+    self.workflow_id = workflow_id
+  end
+
+  def get
```

Comment:
Why we need `validate_and_update_approver_groups` for each `get`? Just want to update the existing `group_refs` attribute to new format?

Reply:
To refresh group name which may be out of sync from rbac.